platform/ios/PolyPodApp/FeatureContainer/domConsole.js

```diff
@@ -1,11 +1,11 @@
+function postLogMessage(message) {
+    window.parent.postMessage(
+        { command: "log", text: JSON.stringify(message) },
+        "*"
+    );
+}
+
 const console = {
-    log(msg) {
-        window.parent.postMessage({ command: "log", text: JSON.stringify(msg) }, "*");
-    },
-    error(msg) {
-        window.parent.postMessage({ command: "log", text: JSON.stringify(msg) }, "*");
-    },
-    dir(object) {
-        window.parent.postMessage({ command: "log", text: JSON.stringify(object) }, "*");
-    }
+    log: postLogMessage,
+    error: postLogMessage,
```

Comment:
You're still eliminating `dir`, which is being used AFAIK

Reply:
OK, no problem, but where? Also, it's the same code as log or error. Can't we just change it to console.log?

Reply:
I think that console.dir imvoked from the webview will not call this one. But it's true that it's called, so I'll leave it there.